### ISSUE TITLE
Provide a ranked source list (studies and trees) for historic synthesis releases (opentree #663)

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/gcmdr#10]
@blackrim @josephwb @kcranston https://github.com/OpenTreeOfLife/opentree/issues/663

If you give me the lists, I can put them in an appropriate form for the webapp to use.

Any metadata you have for them would also be handy, e.g. dates.

**Comment by josephwb** [comment]
Isn't this a table in the manuscript? Lots of metadata there.

**Comment by jar398** [comment]
(1) I don't know where to find the table.
(2) We would like retrospective information - all three trees, not just the
latest.

The longer range goal is to have an ongoing time series of such files, as
further synthetic trees are made. This is just a start.

**Comment by kcranston** [comment]
We should be able to get the information from the commit in this repo that corresponds to the synthesis date, right? @josephwb - what files would we need to look at? Is there a master script that loads the trees from the fungi.py, plants,py, etc scripts?

**Comment by josephwb** [comment]
We haven't used gcmdr for synthesis since before the overhaul of the code
(i.e. the use of subproblems rather than nexsons). Should be able to
reconnect things without too much difficulty.

But I sent you the (python) study lists a short while ago.

Let me know if you need anything else.
JWB.

**Comment by kcranston** [comment]
Yup - I have the lists for v 3.0. I am looking for the lists for v 1.0 and v 2.0. These are the three versions  according to ot10 - list taken from [opentree issue 667](https://github.com/OpenTreeOfLife/opentree/issues/667):

opentree@ashby:/disk2/opentree/downloads$ ls -tl 
total 54069468
-rw-r--r-- 1 opentree opentree 11485911070 May 14 19:19 
treemachine.db.tgz.20150506 
-rw-r--r-- 1 opentree opentree 16226964620 Sep 14 2014 
treemachine.db.tgz.20140914 
-rw-r--r-- 1 opentree opentree 9555496855 Jul 23 2014 
treemachine.db.tgz.20140723

**Comment by josephwb** [comment]
Ok, I'll hunt these down.
JWB.
